app_dart/test/request_handlers/refresh_chromebot_status_test.dart

```diff
@@ -7,22 +7,47 @@ import 'package:cocoon_service/src/model/appengine/task.dart';
 import 'package:cocoon_service/src/request_handlers/refresh_chromebot_status.dart';
```

Comment:
nitty nit: these package:cocoon imports should be between the import package blocks and the relative imports for the test files block

Reply:
Reorganized 😄